test/api_test.js

```diff
@@ -86,4 +86,17 @@ describe('Array', function() {
       assert(response.body.includes('[Papertrail]'));
     });
   });
+
+  describe('bugzilla', () => {
+    const BUGZILLA_BASE_URL = 'https://bugzilla.mozilla.org';
+    const FIRST_BUG_NUMBER = 35;
+
+    it('should fetch the first bug ever filed on bugzilla', async () => {
```

Comment:
I'm not sure this test is useful, as this is testing the API, not any functionality of this tool.